File: Week 6/mario/mario.py
# TODO
from cs50 import get_int

# Get user input
while True:
    height = get_int("Height: ")
    if 0 < height < 9:
        break


for i in range(height):
    # Calculate how many spaces and hashes
    space = height-1 - i
    hashes = i + 1

    # Printing the pyramid
    print(' ' * space + '#' * hashes + '  ' + '#' * hashes)

File: Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_size(void);
void print_pyramid(int n);

int main(void)
{
    int n = get_size();
    print_pyramid(n);
}

//Get pyramid size
int get_size(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    return n;
}


//Print the pyramid
void print_pyramid(int n)
{
    for (int i = 1; i <= n; i++)
    {
        //Calculates how much space is needed in that row
        for (int space = 1; space <= n - i; space++)
        {
            printf(" ");
        }

        //Prints the hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}
